x-pack/plugins/apm/public/components/shared/Links/MachineLearningLinks/useTimeSeriesExplorerHref.ts

```diff
@@ -20,41 +19,31 @@ export function useTimeSeriesExplorerHref({
   serviceName?: string;
   transactionType?: string;
 }) {
-  const { core } = useApmPluginContext();
+  // default to link to ML Anomaly Detection jobs management page
+  const {
+    core,
+    plugins: { ml },
+  } = useApmPluginContext();
   const location = useLocation();
   const { time, refreshInterval } = getTimepickerRisonData(location.search);
```

Comment:
Same as above:

```ts
const { urlParams } = useUrlParams();
const { rangeFrom, rangeTo, refreshInterval } = urlParams;
```

and then replace 
```
      timeRange: time,
      refreshInterval: refreshInterval as RefreshInterval,
      zoom: time,
```

with:

```ts
      timeRange: { rangeFrom, rangeTo },
      refreshInterval: refreshInterval as RefreshInterval,
      zoom: { rangeFrom, rangeTo },

Reply:
Updated here https://github.com/elastic/kibana/pull/78209/commits/c890c97a8f12a9a3cae5176a24e0bf86dfbb20ed